src/gui/guiFormSpecMenu.cpp

```diff
@@ -1776,6 +1864,57 @@ void GUIFormSpecMenu::parseAnchor(parserData *data, const std::string &element)
 			<< "'" << std::endl;
 }
 
+bool GUIFormSpecMenu::parseStyle(parserData *data, const std::string &element, bool style_type)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() < 2) {
+		errorstream << "Invalid style element (" << parts.size() << "): '" << element
+					<< "'" << std::endl;
+		return false;
+	}
+
+	std::string selector = trim(parts[0]);
+	if (selector.empty()) {
+		errorstream << "Invalid style element (Selector required): '" << element
+					<< "'" << std::endl;
+		return false;
+	}
+
+	StyleSpec spec;
+
+	for (size_t i = 1; i < parts.size(); i++) {
+		size_t equal_pos = parts[i].find('=');
+		if (equal_pos == std::string::npos) {
+			errorstream << "Invalid style element (Property missing value): '" << element
+						<< "'" << std::endl;
+			return false;
+		}
+
+		std::string propname = trim(parts[i].substr(0, equal_pos));
+		std::string value    = trim(unescape_string(parts[i].substr(equal_pos + 1)));
+
+		std::transform(propname.begin(), propname.end(), propname.begin(), ::tolower);
+
+		StyleSpec::Property prop = StyleSpec::GetPropertyByName(propname);
+		if (prop == StyleSpec::NONE) {
+			errorstream << "Invalid style element (Unknown property " << propname << "): '" << element
```

Comment:
The format is valid, but the property is (yet) not. To prevent error floods I think it would be better to make this a `warningstream`, or filter duplicates.